Update Template_VMware_Tools_Windows_performance_counters.xml

w3ich3rt opened the pull request:
update for version 5.0
added graphs, screen and trigger

xenadmin commented:
Sorry I can't merge that. There are three problems I spot on the first sight:
- Several items have a naming like _CS3-something_, is that from your company?
- Many German descriptions, that should be English. 
- IHMO the use of `{HOST.NAME}` in the trigger name is not nice.

w3ich3rt commented:
> Sorry I can't merge that. There are three problems I spot on the first sight:
> 
>     * Several items have a naming like _CS3-something_, is that from your company?
> 
>     * Many German descriptions, that should be English.
> 
>     * IHMO the use of `{HOST.NAME}` in the trigger name is not nice.

oh sh*t - sorry.
I removed the cs3 abbreviation and changed the descriptions to english.
Also I removed the macro in the triggername.

Hope it is correct now :-) if not, just tell.

w3ich3rt commented:
No interest by developers
